samples/buildstrategy/buildkit/buildstrategy_buildkit_cr.yaml

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
Please introduce the `dockerfile` param and use it instead of the inofficial DOCKERFILE.